src/couch_replicator_manager.erl

```diff
@@ -980,3 +990,34 @@ get_json_value(Key, Props, Default) when is_binary(Key) ->
         Else ->
             Else
     end.
+
+
```

Comment:
The following is missing which leads to compilation error about undefined macro `?_test`
```
-ifdef(TEST).
-include_lib("couch/include/couch_eunit.hrl").
```